css/share.css

```diff
@@ -102,7 +69,7 @@ a.showCruds {
 	opacity:.5;
 }
 
-a.unshare {
+.shareWithList > li > a.unshare {
```

Comment:
This is an ID instead of a class in my setup somehow 🤔

Reply:
Seems to also be the case on master. https://github.com/nextcloud/gallery/blob/master/js/vendor/nextcloud/share.js#L208

Reply:
Changing it to `#shareWithList`: 

Before:

<img width="478" alt="bildschirmfoto 2018-03-09 um 11 52 15" src="https://user-images.githubusercontent.com/245432/37204245-7253bd48-2390-11e8-9b28-1786764dc6b8.png">

After:

<img width="490" alt="bildschirmfoto 2018-03-09 um 11 52 47" src="https://user-images.githubusercontent.com/245432/37204250-768616b8-2390-11e8-9d14-18c2adc72dd6.png">